master/buildbot/scripts/upgrade_master.py

```diff
@@ -106,34 +107,38 @@ def upgradeDatabase(config, master_cfg):
 
 
 @in_reactor
-@defer.inlineCallbacks
 def upgradeMaster(config, _noMonkey=False):
     if not _noMonkey:  # pragma: no cover
         monkeypatches.patch_all()
 
     if not base.checkBasedir(config):
-        defer.returnValue(1)
-        return
+        return defer.succeed(1)
```

Comment:
@rutsky I wonder whether the return below
defer.returnValue(1)
was ever executed